include/openmc/mesh.h

```diff
@@ -672,6 +667,12 @@ class UnstructuredMesh : public Mesh {
   //! \return Sampled position within the tetrahedron
   Position sample_tet(std::array<Position, 4> coords, uint64_t* seed) const;
 
+  // Data members
+  double length_multiplier_ {
+    1.0}; //!< Constant multiplication factor to apply to mesh coordinates
+  bool specified_length_multiplier_ {false};
```

Comment:
Seems like we should just check if it's equal to one rather than adding a bool here. It's like having

```
double value {1.0};
bool value_is_one {true};
```

Kinda redundant. Up to you if you want to change that in this PR!

Reply:
Yeah, agreed. I just removed it and instead changed the default length to -1 and then just check if length > 0 as a way of signaling that it was specified.

Reply:
Yay yay, thank you!